x/curating/abci.go

```diff
@@ -44,26 +50,6 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) []abci.ValidatorUpdate {
 				return false
 			})
 
-		rewardAccount, err := sdk.AccAddressFromBech32(post.RewardAccount)
-		if err != nil {
-			panic(err)
-		}
-		creatorVotingPoolReward, err := k.RewardCreatorFromVotingPool(ctx, rewardAccount, qv.VotingPool)
-		if err != nil {
-			panic(err)
-		}
-		emitRewardEvent(ctx, types.EventTypeProtocolReward, types.EventTypeVotingPoolReturn,
```

Comment:
we should remove the event constants from the `types` package if not longer used